scripts/remote-pouch-db-instance.ts

```diff
@@ -0,0 +1,21 @@
+import PouchDB from 'pouchdb';
+import PouchDBFind from 'pouchdb-find';
+import pouchdbHTTPAdapter from 'pouchdb-adapter-http';
+import dotenv from 'dotenv';
+dotenv.config();
+
+import { POUCHDB_DB_URL } from '../src/pouchDB/config';
+
+PouchDB.plugin(PouchDBFind);
+PouchDB.plugin(pouchdbHTTPAdapter);
```

Comment:
The http adapter is not required, so you can remove it.

Reply:
Removed